docs/ert/reference/configuration/syntax.rst

```diff
@@ -0,0 +1,22 @@
+The ert configuration file format
+=================================
+
+The ert config file, workflows, workflow job configuration files and forward
+model configuration files all share the same syntax. These are lines starting
+with one keyword followed by arguments separated by whitespace. e.g.
+
+::
+
+        QUEUE_SYSTEM LOCAL
+        NUM_REALIZATIONS 12
+        -- A comment
+        FORWARD_MODEL DELETE_FILE(<FILES>="file1 file-- file3")
+
+        FORWARD_MODEL TEMPLATE_RENDER( \
+            <INPUT_FILES>=parameters.json, \
+            <TEMPLATE_FILE>=<CONFIG_PATH>/resources/SPE1.DATA.jinja2, \
+            <OUTPUT_FILE>=SPE1.DATA \
+        )
+
+As can be seen, newline can be used inside an argument list by escaping with :code:`\` (like with bash). Arguments
```

Comment:
Think we should also add a warning about whitespace after `/`

Reply:
Added